### ISSUE TITLE
Lab 05 - Missing file structure

### ISSUE THREAD

**Issue by nptfdo** [MicrosoftLearning/AZ-203-DevelopingSolutionsforMicrosoftAzure#94]
## Location

- Course: 05
- Exercise: 02
- Task: 02
- Step[s]: 00

Files not present in  ..Allfiles\Labs\05\Starter\Api

I downloaded the **Allfiles** folder on 5th Nov 2019 and that file structure contains a set of files under Allfiles\Labs\05\Starter\Api\ directory. I over-wrote a couple of these files by mistake, so needed to get a fresh version from GitHub today but looks like the whole file structure is missing from the aforementioned path now.
Also, the code mentioned in Program.cs is different from the code mentioned in the lab 5 answer key.
## Repro steps

1.
1.
1.

**Comment by seesharprun** [comment]
The API folder is empty by design. The last change to the folder and its contents occured last July.

**Comment by nptfdo** [comment]
The folder had files up until last week when I cloned the Allfiles directory. Please refer to the lab 5 answer key document to see references to the file structure.

**Comment by seesharprun** [comment]
I looked at the Answer Key document and saw that you are asked to create a new .NET WebAPI project in the empty API folder (Exercise 2, Task 1). Once you create the project, it generates a Program.cs file that you will edit (Exercise 2, Task 2).

I also checked the GitHub commits. The last change to this repository occurred on October 13. The last change to the AllFiles\Labs\05\Starter\Api folder occured on July 3.
